reactor3/build.gradle

```diff
@@ -1,3 +1,4 @@
 dependencies {
     api libs.reactor.core
+    api libs.context.propagation
```

Comment:
Ditto - can it be `implementation` or should it stay with `api`?

Reply:
Thanks for your comments 🙇‍♂️ 
I think `implementation` would be okay, because `context.propagation` is only used on test codes!